docs/src/appendix/sncast/account/create.md

```diff
@@ -13,6 +13,11 @@ Required.
 
 Account name under which account information is going to be saved.
 
+## `--type, -t <ACCOUNT_TYPE>`
+Optional.
```

Comment:
this might have to be changed depending on where we land with type flag for custom class hashes

Reply:
yes, let's do that :+1: